archiver.go

```diff
@@ -423,6 +423,8 @@ func NewArchiverWorkerPool(
 			return err
 		}
 
+		defer lsess.Close()
```

Comment:
Close can throw an error? maybe we should check it.

Reply:
Yes, you should use `defer ioutil.CheckClose(lsess, &err)` instead.